packages/flutter/test/widgets/editable_text_cursor_test.dart

```diff
@@ -724,6 +724,6 @@ void main() {
       matchesGoldenFile('editable_text_test.2.0.png'),
     );
     debugDefaultTargetPlatformOverride = null;
-  }, skip: !Platform.isLinux);
+  }, skip: identical(0, 0.0) || !Platform.isLinux);
```

Comment:
we should factor out this test into some constant somewhere, like isCompiledToJS or something

Reply:
also, we should factor out Platform in general, so that we have an equivalent of `!Platform.isLinux` that is true on Web and false on Linux.